pkg/operator/controller/canary/controller.go

```diff
@@ -195,9 +195,10 @@ func (r *reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 
 // Config holds all the things necessary for the controller to run.
 type Config struct {
-	Namespace   string
-	CanaryImage string
-	Stop        chan struct{}
+	Namespace     string
+	CanaryImage   string
+	CanaryCommand string
```

Comment:
I dont think `CanaryCommand` needs to be a part of this struct if we are hard coding it anyways.

Reply:
This one we need in operator.go to set up the canary controller.

Reply:
> This one we need in operator.go to set up the canary controller.

Could you elaborate on this? If we are hard coding `CanaryHealthcheckCommand` at the top of this file, we can just reference the local constant (rather than take the canary command as an argument in another part of the code). What do you think?

Reply:
I see now.  I will clean it up.

Reply:
Thanks! :grin: